homeassistant/components/webostv/media_player.py

```diff
@@ -115,12 +106,12 @@ async def wrapper(obj, *args, **kwargs):
 class LgWebOSMediaPlayerEntity(MediaPlayerEntity):
     """Representation of a LG webOS Smart TV."""
 
-    def __init__(self, client: WebOsClient, name: str, customize, on_script=None):
+    def __init__(self, client: WebOsClient, name: str, sources, unique_id, on_script):
         """Initialize the webos device."""
         self._client = client
         self._name = name
-        self._unique_id = client.client_key
-        self._customize = customize
+        self._unique_id = unique_id
+        self._sources = sources
         self._on_script = on_script
 
         # Assume that the TV is not paused
```

Comment:
We need to disconnect the dispatch helper connection below in `async_added_to_hass`. Use `self.async_on_remove`.